torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4843,6 +4854,17 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            supports_forward_ad=True,
            sample_inputs_func=sample_inputs_binary_pwise),
     OpInfo('sub',
+           variant_test_name='without_alpha',
```

Comment:
Same question about sub: shouldn't it just be one OpInfo?

Reply:
Should be resolved in the recent commit. Thanks!